app/views/home/tambahObat.php

```diff
@@ -1,116 +1,167 @@
+                    </div>
+                </div>
+                <div class="mt-2">
+                    <label class="form-label" for="efek-samping">efek samping</label>
+                    <textarea maxlength="65535" name="efek_samping" id="efek_samping"
+                        class="<?=$data['format']['efek_samping']?>" placeholder="efek samping"
+                        value="<?= $data['obat']['efek_samping'] ?>" required></textarea>
+                    <?php if(!empty($data['errors']['efek_samping'])) { ?>
+                    <div class="invalid-feedback ">
+                        <?php echo $data['errors']['efek_samping']; ?>
+                    </div>
+                    <?php } ?>
+                </div>
+                <div class="mt-2">
+                    <label class="form-label" for="indikasi">indikasi</label>
+                    <textarea maxlength="65535" type="text" name="indikasi" id="indikasi"
+                        class="<?=$data['format']['indikasi']?>" placeholder="indikasi"
+                        value="<?= $data['obat']['indikasi'] ?>" required></textarea>
+                    <?php if(!empty($data['errors']['indikasi'])) { ?>
+                    <div class="invalid-feedback ">
+                        <?php echo $data['errors']['indikasi']; ?>
+                    </div>
+                    <?php } ?>
+                </div>
+                <div class="mt-2">
+                    <label class="form-label" for="kontradiksi">kontradiksi</label>
+                    <textarea maxlength="65535" type="text" name="kontradiksi" id="kontradiksi"
+                        class="<?=$data['format']['kontradiksi']?>" placeholder="kontradiksi"
+                        value="<?= $data['obat']['kontradiksi'] ?>" required></textarea>
+                    <?php if(!empty($data['errors']['kontradiksi'])) { ?>
+                    <div class="invalid-feedback ">
+                        <?php echo $data['errors']['kontradiksi']; ?>
+                    </div>
+                    <?php } ?>
+                </div>
+                <div class="mt-2">
+                    <label class="form-label" for="peringatan">peringatan</label>
+                    <textarea maxlength="65535" type="text" name="peringatan" id="peringatan"
+                        class="<?=$data['format']['peringatan']?>" placeholder="peringatan"
+                        value="<?= $data['obat']['peringatan'] ?>" required></textarea>
+                    <?php if(!empty($data['errors']['peringatan'])) { ?>
+                    <div class="invalid-feedback ">
+                        <?php echo $data['errors']['peringatan']; ?>
+                    </div>
+                    <?php } ?>
+                </div>
+                <div class="mt-2">
+                    <label class="form-label" for="interaksi-obat">interaksi obat</label>
+                    <textarea maxlength="65535" type="text" name="interaksi_obat" id="interaksi_obat"
```

Comment:
`for` di label tidak sesuai dengan `id` di input.
input tidak bisa fokus ketika klik label